### ISSUE TITLE
Created a milight platform

### ISSUE THREAD

**Issue by normen** [shamasis/homebridge-milight-hub#5]
Hey,

I created a milight-hub platform plugin for homebridge that uses auto-discovery of lights and optionally MQTT, maybe you want to integrate some of the code or join the plugins somehow. I just created it because I needed a solution that didn't require me to configure everything twice.

https://github.com/normen/homebridge-milighthub-platform

Just thought I would mention.

Cheers,
Normen

**Comment by shamasis** [comment]
@normen ... I'm glad you posted here. You have got a very interesting plugin. Does that plugin not require Milight Wifi Bridge? This plugin, is meant to complement the DIY ESP32/8266 based hub. I am a bit confused as to how both can interoperate.

**Comment by normen** [comment]
Nah, my plugin is for the milight-hub (hence MQTT). I just replaced a bunch of Mi-Boxes with that and was sick of entering everything thrice.. 😄 So thats the main reason for my plugin - the auto-discovery of aliases in milight-hub based on HomeBridge dynamic platform.

It works for my purposes, I just fixed the "HomeKit sends all values separately" issue. Theres no support for step-wise dimming of white lights, I only own RGB(W) strips and bulbs.

So yeah, I just noticed that theres a bunch of MiLight plugins all trying to solve various similar issues so I thought I'd mention my new addition to that zoo 😉
